kubernetes/spack/internal-spack-io/gh-gl-sync/cron-jobs.yaml

```diff
@@ -13,22 +15,22 @@ spec:
           restartPolicy: Never
           containers:
           - name: sync
-            image: alpine
-            imagePullPolicy: IfNotPresent
-            command: ["sh", "/scripts/entrypoint.sh"]
-            volumeMounts:
-              - name: scripts
-                mountPath: "/scripts"
-              - name: secrets
-                mountPath: "/secrets"
-          volumes:
-            - name: scripts
-              configMap:
-                name: gh-gl-sync
-                defaultMode: 0700
-            - name: secrets
-              secret:
-                secretName: gh-gl-sync
-                defaultMode: 0700
+            image: zackgalbreath/spack-ci-bridge:latest
```

Comment:
Considering the change in service limits on dockerhub, consider switching to a static tag and `imagePullPolicy: IfNotPresent`.

Reply:
Good suggestion, I've made this change.